.jshintrc

```diff
@@ -17,7 +17,6 @@
   "browser": true,
   "globals": {
     "module": true,
-    "angular": true,
-    "_": true
+    "angular": true
```

Comment:
added `"esversion": 6` and `"globals": [ "define": true, "require": true]`. its lint free now